### DISCUSSION TITLE
Tracking notification.

### DISCUSSION THREAD

**Discussion by LelandSindt** [teslamate-org/teslamate#3357] [General]
https://tesla-info.com/release/2023.38

``` An icon appears at the top of your touchscreen when an app requests your vehicle’s live location data. ```

I would imagine that the default behaviour of TeslaMate is going to cause this icon to appear constantly. 

Anyone see this in the wild yet?

**Comment by arter97** [comment]
I've just updated my Model 3 to 2023.38.9.

Teslamate was updated to the latest version prior to the car update, so mine didn't break completely but now the location icon is flashing constantly.

It would be understandable to have it constantly on or off, but it's intermittently turning on and off, which is very annoying.

Is there a possible fix for this? I'm yet to hear a similar story.

**Reply by LelandSindt** [reply]
I doubt there is anything that we can do on the TeslaMate side.

I would be curious to know if/how the behaviour of the tracking icon changes with `Streaming API` enabled/disabled.

**Reply by timvdhoorn** [reply]
> I doubt there is anything that we can do on the TeslaMate side.
> 
> I would be curious to know if/how the behaviour of the tracking icon changes with `Streaming API` enabled/disabled.

Sorry to bumb, but is the behaviour of the tracking icon changed with enabling or disabling `Streaming API`?

**Reply by LelandSindt** [reply]
@timvdhoorn I don't know for sure... I have not experimented with it. However, I assume that disabling the `Streaming API` would cause the tracking icon to show up less often. The trade off as mentioned in #3970 is that Tesla is putting rate limits on their API that you will likely quickly exhaust as Teslamate falls back to the API if you disable streaming. 

I have learned to ignore the icon. ---- Its there to let the driver know that "someone is watching you" in this case, that someone is Teslamate...

**Comment by LelandSindt** [comment]
I got the update a while back and can confirm that Teslamate causes the icon to appear as expected... Again I doubt there is anything that can be dome to disable it or influence it to constantly or intermittently display.

**Comment by nagyrobi** [comment]
The flashing icon is pretty annoying...
I tested ABRP too, enabled streaming API connection to it, and the icon remains constantly on. Afaik ABRP [uses Teslamate under the hood](https://github.com/fetzu/teslamate-abrp) to access the car stream. Wonder what did they do to their Teslamate instance to avoid icon flashing?
(maybe they don't re-connect periodically but stream constantly?)
